readthedocs/builds/models.py

```diff
@@ -1030,3 +1046,10 @@ def match(self, version, match_arg):
         except Exception as e:
             log.info('Error parsing regex: %s', e)
             return False, None
+
+    @property
+    def edit_url(self):
```

Comment:
It would be good if we follow the convention (`get_absolute_url`, `get_docs_url`, etc) we already have naming this `get_edit_url` and not making it a property.